### DISCUSSION TITLE
How to access the @db namespace inside a TS file

### DISCUSSION THREAD

**Discussion by kelvin2200** [prisma/prisma#10757] [Q&A]
Is there some package that I can import for things like `@db.UnsignedInt`.
The prisma vscode extension has an autocomplete feature for that inside `.prisma` files.
How can I use that in a normal javascript/TS file?

**Comment by janpio** [comment]
What do you need that for in a JS/TS file? Usually the native types are only used in the Prisma Schema file - where the VSCode extension helps with auto completion and so on.

**Reply by kelvin2200** [reply]
I am building a prisma schema generator for a code-first approach type of project that uses prisma. The generator takes some classes with some decorators and outputs a `.prisma` file. I need to be able to decorate fields in those classes with `@db` properties, and i don't want to forge them.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
